ProposedChangesToSpecification.md

```diff
@@ -30,3 +30,145 @@ for storing B1+ fielmaps.
+* `Grouping suffixes` MUST be used in conjuction with at least one of the `<indexable_metadata>-<index>` and `acq-<label>` key-value pairs. If existing `<indexable_metadata>-<index>` and `acq-<label>` key-value pairs are not enough to describe a new qMRI method, additional request is needed to extend those lists.     
+
+*** 
+
+* `Designation suffixes for qMRI maps` MUST attain a clear description of the parameter that they contain, **including the units**. If available, hyperlinks to example applications and/or more detailed descriptions are highly encouraged.
+
+* If there exist a `designation suffix for qMRI maps` that relates to one or many `grouping suffix`, all the relevances MUST be indicated in the description of the `designation suffix for qMRI map` by the _"Can be generated by: "_ expression.
+
+*** 
+
+* The list of available `suffixes for conventional MRI contrasts` is immutable: `T1w`,`T2w`,`PDw`,`T2starw`. 
+
+***
+
+
+## Indexable Metadata 
+
+### Proposed Change 
+
+### Justification 
+
+## Sidecar JSON files of qMRI maps
+
+Which metadata fields must be included. Some of these fields are to be inherited by the input files and some of them are to be populated by the estimation software. Right now: 
+
+* BasedOn --> List of files gruoped by an grouping suffix + (optional) field maps. 
+* MagneticFieldStrength
+* Manufacturer
+* ManufacturerModelName
+* InstitutionName 
+* PulseSequenceType
+* PulseSequenceDetails
+* EstimationPaper
+* EstimationAlgorithm
+* EstimationSoftwareName
+* EstimationSoftwareVersion 
+* EstimationSoftwareLanguage
+* EstimationSoftwareEnv 
+
+## Sidecar JSON files of grouping suffixes 
+
+Mention about root JSON + repeating JSON file convention.
+Which root JSON metadata fields are optional/required. 
+
+
+## Legacy suffixes for anatomical data
+### Proposed Change 
+Moves some suffixes for `anatomical data` to a new table called "LEGACY suffixes". These suffixes are still valid BIDS format,
+but we reccomend not to use them anymore. They are ambiguous and/or inconsistent.
+
+### Justification 
+The following suffixes are moved to the legacy table:
+ * **`_T2star`**: what is meant is T2\*-_weighted_ data. To be consistent with the suffixes `T1w`, `T2w` and `PDw`, we want
+to use the suffix `_T2starw` for this.
+ * **`_FLAIR`**: This is a sequence name rather than a contrast or a way of grouping quantitative maps. We prefer to use
+`_T2w`, `_PDw`, or `_MESE` depending on the use case.
+ * **`_FLASH`**: This is a sequence name rather than a contrast or a way of grouping quantitative maps. We prefer to use
+`_T2starw`, `_PDw`, or `_GREME` depending on the use case.
+ * **`_PD`**: what is meant is PD*-_weighted_ data. To be consistent with the suffixes `T1w`, `T2w` and `T2starw`, we want
+to use the suffix `_PDw` for this.
+ * **`_PDT2`**: We recommend to use `PDT2map` for this.
```

Comment:
@nicholst - we're not super clear on what the `PDT2` scan _is_ and so we're not super sure about how to classify it! Can you help us? (References fine too!)

Reply:
PD T2 “dual echo” is just something that I’ve always seen with clinical scans.  I found this reference: [Fast Spin Echo Dual Echo](http://mriquestions.com/dual-echo-fse.html), writing

> For the first 20 years of clinical MRI, double-echo sequences were part of standard protocols for nearly all body parts. Short and long echo times (TE) were used to generate proton-density and T2-weighted images in a single acquisition using a long repetition time (TR). Over the last decade T2-FLAIR has largely replaced proton-density sequences for neuroimaging, but they are still used in musculoskeletal and other body applications.

While it may seem like an anachronism this dual echo pair is very common in existing clinical datasets, especially for clinical trials where the protocol was specified long ago.

Reply:
Hi @nicholst!! Thank you so much for commenting here - I'm so sorry that we've dropped this thread for so long.

@agahkarakuzu, @Gilles86 and I are just reviewing this pull request and wondering about whether this scan is _one_ file (maybe with two volumes in it - in the 4th dimension) or two files? Or are the two echos combined in someway to one 3D file?

Thank you!

Reply:
> @agahkarakuzu, @Gilles86 and I are just reviewing this pull request and wondering about whether this scan is one file (maybe with two volumes in it - in the 4th dimension) or two files? Or are the two echos combined in someway to one 3D file?

IMHO, the `PDT2` suffix only makes sense for a 2-volume image... if the images are separated out, then one would be `_PD` and the other `_T2`.  _However_, `heudiconv` presently uses it with 3D images too:  `dcm2nii` creates 2 3D images, but `heudiconv` uses a logic that requires images sharing a DICOM Series UID to share a BIDS base and image type;  so since both the PD and T2 arise from the same series, they get the same base and then must be distinguished with (e.g.) `echo-1` and `echo-2`.

Does this help?